packages/fixtures/package.js

```diff
@@ -0,0 +1,15 @@
+Package.describe({
+  name: 'mantle:fixtures',
+  version: '0.0.1',
+  debugOnly: true,
+  summary: '',
+  git: '',
+  documentation: 'README.md'
+});
+
+Package.onUse(function(api) {
+  api.versionsFrom('1.1.0.2');
+  api.use('useraccounts:core');
```

Comment:
This could be accounts-base - we don't need any of the things useraccounts adds.